packages/next/pages/_document.tsx

```diff
@@ -13,7 +13,6 @@ import {
   DocumentInitialProps,
   DocumentProps,
 } from '../next-server/lib/utils'
-import fidPolyfill from '../next-server/lib/fid'
```

Comment:
Do you want to remove this? The `web-vitals` library will use it if it's present (but it doesn't add it itself).

Reply:
Ahh, nevermind, I see you mention this specifically in the PR description.

Reply:
Yeah polyfill was behind a flag until now. Doesn't make as much sense to let users know to flip the flag when they can just include it themselves if they need to.